spikeinterface_gui/unitlist.py

```diff
@@ -274,7 +282,7 @@ def on_visible_shortcut(self):
         for row in rows:
             self.table.selectRow(row)
 
-    def delete_unit(self):
+    def delete_unit(self, event=None):
```

Comment:
event is panel only no ?

Reply:
It's needed so we can call the exact same function. Otherwise we need to make a wrapper for panel that takes an event